recipe/meta.yaml

```diff
@@ -23,7 +23,7 @@ requirements:
     - pip
   run:
     - python ==2.7.*|>=3.3
-    - dash
+    - dash ==2.0.0
```

Comment:
I think you will get a ciruclar dependency issue in the conda-forge set of packages if you pin dash here and dash-table on the dash feedstock. That's why these are left unpinned.

Reply:
That is true normally! But my update to the dash feedstock actually took these dependencies away as they are integrated into dash now. And dash-table now requires dash 2.0.0 since it imports from that. Does that help?

Reply:
Good question. I think if we merge these in, then they'll be done from what I can see in dash documentation. Looks like these would be the last releases for these packages anyway.

Reply:
I've closed and reopened these in order to trigger the builds again. Let me know what you think!